src/controllers/main.controller.ts

```diff
@@ -1,11 +1,12 @@
 import { Request, Response } from "express";
 
-export class MainControler {
+class MainControler {
```

Comment:
No classes

Reply:
And you don't need to name the file `main.controller.ts`, `main.js` or `main.ts` is fine